Advanced-Recruit-Training/src/BatteryStateDisplayService.cpp

```diff
@@ -24,10 +24,12 @@ BatteryStateDisplayService::~BatteryStateDisplayService()
 void BatteryStateDisplayService::handleBatteryDataReceived(const BatteryData& batteryData)
 {
     batteryStateOfChargeService_.addData(batteryData);
-
+/*
     QTextStream(stdout) << "Voltage: " << batteryData.voltage
         << " Current: " << batteryData.current
-        << " Total Ah used: " << batteryStateOfChargeService_.totalAmpHoursUsed() << endl;
-
+        << " Total Ah used: " << batteryStateOfChargeService_.totalAmpHoursUsed()
+    //QString text =QTime::currentTime().toString();
+        << "TIME UNTIL BATTERY FULLY CHARGED" << batteryStateOfChargeService_.timeWhenChargedOrDepleted().toString()<<endl;
+*/
```

Comment:
In general, commented code should be deleted in a PR. That's what version control is for.